SE-XXX Typed throws.md

```diff
@@ -455,7 +490,13 @@ public func foo() throws MyError {
     throw MyError.baz 
   }
 }
+```
+
+### Multiple throwing types in the same do statement

```

Comment:
I really have big concerns implementing it like this. I guess it needs to be evaluation first how would look with swift error conversions supported by swift.

Reply:
Yeah, we need a primary feedback from the reviewer on the topic and then we can update in a return for revision